src/Razor/src/Microsoft.VisualStudio.Editor.Razor/LegacyProjectEngineFactory_3_0.cs

```diff
@@ -29,13 +29,6 @@ public RazorProjectEngine Create(RazorConfiguration configuration, RazorProjectF
 
                 initializer.Initialize(b);
                 configure?.Invoke(b);
-
-                // See comments on MangleClassNames
-                var componentDocumentClassifier = b.Features.OfType<ComponentDocumentClassifierPass>().FirstOrDefault();
-                if (componentDocumentClassifier != null)
-                {
-                    componentDocumentClassifier.MangleClassNames = true;
```

Comment:
Can we also remove support for `MangleClassNames`?

Reply:
Yeah but there are some places it is used like in `DefaultTagHelperBinderPhase` so it will take some time to make sure removing it doesn't break that code. I'm going to address this as part of a new PR targeting preview3.